### ISSUE TITLE
Store DICOM field (0018,9093) NumberOfKSpaceTrajectories ?

### ISSUE THREAD

**Issue by baxpr** [rordenlab/dcm2niix#844]
Feature request. We've got a custom MP2RAGE sequence running on Philips where this field 

    (0018,9093) NumberOfKSpaceTrajectories

stores a value that's needed for an accurate T1 estimation. It would be helpful to have it in the BIDS sidecar. No idea if this would be useful beyond this specific case though.

**Comment by neurolabusc** [comment]
@baxpr I would suggest you make your own fork of dcm2niix. Alternatively, propose this as a new key value pair for the BIDS specification. Historically, I tried to add a lot of useful tags. However, when the ASL tags were formally added to the BIDS specification different names were provided than those that I used (which were taken from the manufacturer). Therefore, moving forward I am reluctant to add tags unilaterally.
